### DISCUSSION TITLE
How to create a coil

### DISCUSSION THREAD

**Discussion by 784622315** [simnibs/simnibs#345] [2. General]
Dear expert, I would like to ask what is the "segment_count" in the coil program in the example? Is it related to the number of turns of the coil? Also, how can I add the number of turns and inductance of the coil to the program? Thank you for your answer.

**Comment by oulap** [comment]
Hi,

We're currently updating the documentation and examples related to the coil creation code. This is hopefully done in the near future. I'll ping this discussion once the documentation is there. Sorry to keep you waiting.
